lib/components/SpeedkitVimeo/Base.vue

```diff
@@ -152,7 +152,8 @@ export default {
           sizes: this.posterSizes,
           media: 'all'
         }],
-        loadingSpinner: this.posterLoadingSpinner
+        loadingSpinner: this.posterLoadingSpinner,
+        crossorigin: 'anonymous'
```

Comment:
why do we need a crossorigin attribute at the loading spinner? the spinner resource should be inlined else we have additional roundtrips

Reply:
Will be removed, makes no sense because the resources are retrieved locally loaded. Question is whether the crossorigin can be customized by the poster or can only be defined per module option.